src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -52,24 +54,55 @@ case class WorkflowDescriptor(id: WorkflowId,
   val shortId = id.toString.split("-")(0)
   val name = namespace.workflow.unqualifiedName
   val actualInputs: WorkflowCoercedInputs = coercedInputs ++ declarations
-  private val props = sys.props
   private val relativeWorkflowRootPath = s"$name/$id"
   private val log = WorkflowLogger("WorkflowDescriptor", this)
-  val workflowOutputsPath = workflowOptions.get("outputs_path") recover { case e: IllegalArgumentException =>
-    log.warn("outputs_path expected to be of type String", e)
+  val workflowLogDir = workflowOptions.get(WorkflowLogDirOptionKey) recover { case e: IllegalArgumentException =>
+    log.warn(s"$WorkflowLogDirOptionKey expected to be of type String", e)
+    throw e
+  }
```

Comment:
Nope. These objects are of type `Try[String]`. The exception is logged once here, and then the object is pretty much treated by callers as an `Option[String]` equal to `None`. They generally call `map` with at `getOrElse`, or `match` on the `Failure[_]` ignoring the error.